src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -56,7 +66,15 @@ public static Logger getLogger(String name) {
      */
     public static <T> Logger getLogger(Class<T> clazz) {
         requireNonNull(clazz);
-        return getLogger(clazz.getSimpleName());
+        // The BASE_LOGGER_NAME acts as a common prefix for all application loggers,
+        // organizing them based on functionality or module.
+        // The dot (".") serves as a separator, reflecting the package structure and providing a
+        // hierarchical representation.
+
+        // Changing the BASE_LOGGER_NAME affects all related loggers, enabling centralized control
+        // over their configuration and behavior. It allows for easy adjustment of multiple loggers
+        // simultaneously.
+        return getLogger(BASE_LOGGER_NAME + "." + clazz.getSimpleName());
```

Comment:
Something like this?
```
Java organizes loggers into a hierarchy based on their names (using '.' as a separator, 
similar to how Java packages form a hierarchy). Furthermore, loggers without a level inherit the
level of their parent logger.
By prefixing names of all loggers with baseLogger's name + ".", we make the 
baseLogger the parent of all loggers. This allows us to change the level of all
loggers simply by changing the baseLogger level.
```

Reply:
Separately, you used `//` for this block comment, but in another PR you used `/* ... */`. Let's use one style consistently. I think `//` is preferred.